src/Dfc.CourseDirectory.Database/Pttcd/Views/vwProviderSearchIndex.sql

```diff
@@ -3,11 +3,19 @@ AS
 SELECT      p.ProviderId,
             p.Ukprn,
             p.ProviderName,
+            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
+            pc.AddressPostcode Postcode,
             p.ProviderStatus,
             p.UkrlpProviderStatusDescription,
-            pc.AddressPostcode Postcode,
-            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
-            p.Version -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+            p.Version, -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
+
+            -- additional fields to skew the search index score approximately as per the old index
+            coalesce(p.CourseDirectoryName,p.ProviderName) CourseDirectoryName, -- as per previous cosmos-based search index definition
```

Comment:
Use CAPS for function names, space after commas for readability

Reply:
done, thanks